src/lib/journey/callbacks/boolean/boolean.svelte

```diff
@@ -37,11 +40,10 @@
 
 <Checkbox
   errorMessage={validationFailure}
-  {isRequired}
   isInvalid={!!validationFailure}
   key={inputName}
   onChange={setValue}
   value={previousValue}
 >
-  {prompt}
+  {interpolate(outputName.replace('/', ''), null, prompt)}
```

Comment:
You'll notice that all the changes to the callback handlers is essentially the use of this `interpolate` function. Which replaces hard coded strings.

Reply:
I left a comment on [src/locales/ca/en/index.json](https://github.com/cerebrl/forgerock-web-login-framework/pull/6/files/2a6df6259179921f4523db6c4f9c33c287df8736#diff-0b04e3700cfa2909235f08372cab26546edbb56b662609ac31e76de730052a6a) as it is still a little unclear how some of these strings are being used, e.g. some seem to be callback types, some seem to be based on the prompts configured in AM. Where are the callback type names being interpolated?